<?xml version="1.0"?>
<launch>
    <node pkg="velodyne_pointcloud" type="cloud_node" name="convert_veloscans">
	<param name="calibration" value="$(find ndt_fuser)/launch/velo32.yaml" />
    </node>
    <node pkg="ndt_fuser" type="ndt_fuser" name="laser_fuser_2d" output="screen">
	<param name="points_topic" value="/velodyne_points" />
	<param name="matchLaser" value="false" />
	<param name="size_x_meters" value="40" />
	<param name="size_y_meters" value="40" />
	<param name="size_z_meters" value="2" />
	<param name="resolution" value="0.5" />
	<param name="sensor_range" value="30." />
	<param name="sensor_pose_x" value="0.115" />
	<param name="beHMT" value="true" />
	<param name="match2D" value="true" />
	<param name="useOdometry" value="true" />
	<param name="odometry_topic" value="/vmc_navserver/odom" />
	<param name="visualize" value="true" />
    </node>
</launch>
